Mixpanel/Mixpanel.m

```diff
@@ -315,17 +321,21 @@ - (void)identify:(NSString *)distinctId
                 self.alias = nil;
                 self.distinctId = distinctId;
             }
-            self.people.distinctId = distinctId;
-        }
-        if (self.people.unidentifiedQueue.count > 0) {
-            for (NSMutableDictionary *r in self.people.unidentifiedQueue) {
-                r[@"$distinct_id"] = self.distinctId;
-                @synchronized (self) {
-                    [self.peopleQueue addObject:r];
+            if(usePeopleDistinctId) {
```

Comment:
why is this inside the `if` condition starting at line 319?
Shouldn't the queued updates for `people` be processed even if the provided distinctId is equal to the stored alias @pchien ?